app/views/layouts/_menu.html.erb

```diff
@@ -4,17 +4,17 @@
       <li class="dropdown">
         <%= menu_dropdown_link(t(:administrate)) %>
         <ul class="dropdown-menu">
-          <% if can? :manage, Vote %>
+          <% if can? :index, Vote %>
```

Comment:
That makes sense - if we still use `:manage` for everything it wouldn't manage.
If we want to use single action admin-permissions we need to rethink the menu system.